### ISSUE TITLE
Transparent `customBottomWidget`

### ISSUE THREAD

**Issue by noaonoszko** [flyerhq/flutter_chat_ui#582]
Is it possible to have a transparent `customBottomWidget`? It seems impossible to me. I did an experiment in which I changed the `backgroundColor` in the theme. Then my custom bottom widget changed color as well. But when I scroll, the bottom widget obscures the messages.

I think this would be a quite nice feature. The Messenger and the iPhone Messages app both have transparent input widgets.

**Comment by demchenkoalex** [comment]
Hey, can you show how this feature looks?

**Comment by noaonoszko** [comment]
Here's what it looks like in Messenger, which has a somewhat transparent bottom widget. It can be seen in blue. ![image](https://github.com/flyerhq/flutter_chat_ui/assets/47183162/0c152023-fa46-491a-9a47-f3148ec95b7a)

I realized that the issue might not be about opaqueness, but instead that the bottom widget is below the chat messages area and not stacked on it as I initialy thought. 

Lastly, I fixed my typo customButtonWidget to customBottomWidget.
